src/pages/develop/services.md

```diff
@@ -14,7 +14,7 @@ A railway service is a deployment target for your application. There are two typ
 
 Deployment services can be connected to a GitHub repo and autodeployed on each commit. Templates are a pre-configured groups of services that can be used as a starting off point for your project, or added to an existing project).
 
-Services keep track of all current and historical deployment attempts as well as [variables](/develop/deployments), [metrics](/diagnose/metrics), and application source like a GitHub repo and a start command. Clicking a service within the project canvas will bring up the service view on the side of your browser.
+Services keep track of all current and historical deployment attempts as well as [variables](variables), [metrics](/diagnose/metrics), and application source like a GitHub repo and a start command. Clicking a service within the project canvas will bring up the service view on the side of your browser.
```

Comment:
Shouldn't this be `/develop/variables`?

Reply:
hey @FarazPatankar not sure we need the full path in this case, but happy to add